src/Transition/CreateNewIDFUsingRulesV24_1_0.f90

```diff
@@ -382,7 +382,18 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
 !                 CurArgs = CurArgs + 1
 
               ! If your original object starts with A, insert the rules here
-
+              CASE('AIRLOOPHVAC:UNITARYSYSTEM') ! add new input field No Load Supply Air Flow Rate Control Set To Low Speed
+                  CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                  nodiff=.false.
+                  OutArgs(38)=InArgs(38)
+                  ! OutArgs(39) is added. If existing coil is VS DX cooling or heating coil then set YES, otherwise set NO
+                  IF ( SameString(InArgs(12),'Coil:Heating:DX:VariableSpeed') .or. SameString(InArgs(15),'Coil:Cooling:DX:VariableSpeed') ) THEN
+                      OutArgs(39) = 'YES'
```

Comment:
@mjwitte 2 things: does this argument need to be lower case "Yes" to match the idd?, and the comment is wrong here and below, should be OutArgs(10) is added.

    A7 , \field No Load Supply Air Flow Rate Control Set To Low Speed
        \type choice
        \key Yes
        \key No

Reply:
> @mjwitte does this argument need to be lower case "Yes" to match the idd?

Not required, but looks nicer.